app/lib/filterServices.js

```diff
@@ -4,7 +4,7 @@ const getOpeningHoursMessage = require('../lib/getOpeningTimesMessage');
 const utils = require('../lib/utils');
 
 function sortByDistance(a, b) {
-  return a.distanceInMiles - b.distanceInMiles;
+  return a.dist - b.dist;
```

Comment:
This doesn't need to be done if the query is changed to `distanceField: 'distanceInMiles'`
https://github.com/nhsuk/nearby-services-api/pull/30/files#diff-6b3c91a372d0bb755710f47f6b10c6e3R20